ocfweb/account/register.py

```diff
@@ -107,6 +109,23 @@ def request_account(request):
         },
     )
 
+def recommend(request):
+    real_name = request.GET.get('real_name', '')
+    first_name, last_name = real_name.split()
+    rec_lst = recommender.recommend(first_name, last_name, 3)
+    rec_dict = {}
+    for i in range(len(rec_lst)):
+        rec_dict[str(i)] = rec_lst[i] # convert i to string because JS dicts need string keys
```

Comment:
I don't think you need to do this conversion manually, the JsonResponse function should automatically convert this dict to valid JSON. Also, these variables should probably be named properly

Reply:
As in the keys of the dictionary? I was intending this to essentially mirror the surface functionality of an array or a list. But if the potential security concern I listed above is to be disregarded, then I'll just switch to a list which'll be much cleaner.